packages/x-charts/src/constants/index.ts

```diff
@@ -13,3 +13,8 @@ export const DEFAULT_AXIS_SIZE_HEIGHT = 25;
 
 // How many pixels to add to the default axis size if that axis has a label
 export const AXIS_LABEL_DEFAULT_HEIGHT = 20;
+
+export const ANIMATION_DURATION_MS = 300;
+// Similar to d3-ease's easeCubicInOut.
+// Comparison: https://codesandbox.io/p/sandbox/8732px
+export const ANIMATION_TIMING_FUNCTION = 'cubic-bezier(0.66, 0, 0.34, 1)';
```

Comment:
An unfortunate consequence of having a hybrid JS/CSS animation system is that we need to make sure they look similar. For example, the line chart will have its line animated using JS and mark elements using CSS.

As such, I searched for a CSS timing function that is similar to `d3-ease`'s `easeCubicInOut`. 

Created [this CodeSandbox](https://codesandbox.io/p/sandbox/8732px) to compare them. 

Unfortunately, we can't use CSS's `easeInOut` since that causes the animation to be out of sync in some cases.

With `easeInOut`:


https://github.com/user-attachments/assets/a32c3b20-96c5-4e4f-ab03-9d588ce757c7





With this custom animation timing function:

https://github.com/user-attachments/assets/2bf223e9-5fe1-4a61-a67f-d91c06aec099

Reply:
Can't we just use `const cubicBezier = BezierEasing(0.66, 0, 0.34, 1);` as the easing function in D3? This way both css and d3 would be the same.

Reply:
Yeah, I was trying to avoid adding too many dependencies, but I think that works:

https://github.com/user-attachments/assets/e47420a8-fb86-4385-8504-d7089da74cbf